discojs/src/aggregator/base.ts

```diff
@@ -69,14 +69,34 @@ export abstract class Base<T> extends EventEmitter<{'aggregation': T }> {
 
   /**
    * Adds a node's contribution to the aggregator for the given aggregation and communication rounds.
+   * The aggregation round is increased whenever a new global model is obtained and local models are updated.
+   * Within one aggregation round there may be multiple communication rounds (such as for the decentralized secure aggregation
+   *  which requires multiple steps to obtain a global model)
    * The contribution will be aggregated during the next aggregation step.
    * @param nodeId The node's id
    * @param contribution The node's contribution
-   * @param round For which aggregation round the contribution was made
-   * @param communicationRound For which communication round the contribution was made
+   * @param round aggregation round of the contribution was made
+   * @param communicationRound communication round the contribution was made within the aggregation round
+   * @returns boolean, true if the contribution has been successfully taken into account or False if it has been rejected
    */
   abstract add (nodeId: client.NodeID, contribution: T, round: number, communicationRound?: number): boolean
 
+  /**
+   * Evaluates whether a given participant contribution can be used in the current aggregation round
+   * the boolean returned by `this.add` is obtained via `this.isValidContribution`
+   */
+  isValidContribution(nodeId: client.NodeID, round: number): boolean {
+    if (!this.nodes.has(nodeId)) {
+      console.warn("Contribution rejected because node id is not registered")
```

Comment:
we can use `debug` now instead :)

```suggestion
      debug("Contribution rejected because node id is not registered")
```

Reply:
Ah yes that's left from when I couldn't manage to print debug statements in the browser